web/pom.xml

```diff
@@ -60,9 +51,19 @@
       <artifactId>applicationinsights-runtime-attach</artifactId>
       <version>3.3.1</version>
     </dependency>
+    <!-- Spring Cloud Azure dependencies -->
+    <dependency>
+      <groupId>com.azure.spring</groupId>
+      <artifactId>spring-cloud-azure-starter-jdbc-postgresql</artifactId>
+    </dependency>
+    <!-- Test dependencies -->
+    <dependency>
```

Comment:
we can remove this test dependency since we don't have any test at the moment

Reply:
Will do.